### ISSUE TITLE
[BUG] In-file search won't open in Reading view

### ISSUE THREAD

**Issue by Sinled** [scambier/obsidian-omnisearch#222]
**Problem description:**

Maybe i am missing something, but **in file** search panel won't open when editor is in **reading view**, Vault search - works fine, 
in source mode, and live preview it is also working fine.

There are no errors in console

**Your environment:**

- Omnisearch version: 1.13
- Obsidian version: 1.1.16
- Operating system: Mac os 12.6
- Number of indexed documents in your vault (approx.): 5

**Things to try:**

- Does the problem occur when Omnisearch is the only active community plugin:  Yes
- Does the problem occur when you _don't_ index PDFs, images, or other non-notes files:  Yes
- Does the problem occur after a cache reset: Yes

i've also tried 
- Enabling Default theme
- disabling vim keybindings

**Comment by scambier** [comment]
Yep, In-File search doesn't work in reading mode. Not really a bug, just something I never implemented 👍

Related to #173
